<!-- Base Interfaces (no controllers or arduino) launch file -->
<launch>
  <!-- Boat Interface nodes -->
  <rosparam file ="$(find boat_interfaces)/config/parameters.yaml" command="load" />
  <node pkg="boat_interfaces" type="boat_state_controller_node.py"
    name="boat_state_controller">
  </node>

  <node pkg="boat_interfaces" type="rudder_node.py"
    name="rudder">
  </node> 

  <node pkg="boat_interfaces" type="winch_node.py"
    name="winch" >
  </node>

  <node pkg="boat_interfaces" type="tacking_action.py"
    name="tacking_action" output="screen">
  </node>
  
  <node pkg="boat_interfaces" type="pid.py"
    name="controller" ns="rudder_pid" output="screen">
  </node> 
</launch>
